concatenate_ice/concat_ice_daily.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+#concatenate sea-ice daily output
+#script inspired from https://github.com/COSIMA/1deg_jra55_ryf/blob/master/sync_data.sh#L87-L108
+
+out_dir=$(ls -td archive/output??? | head -1)/ice/OUTPUT #latest output dir only
```

Comment:
It would be good to default to the most recent output directory, but allow the user to specify a directory as a command line input. I'm thinking about the use case where someone forgets to turn on `sync` and then wants to manually concatenate the files in a bunch of directories.  

We can leave it up to the user to create a loop to call it multiple times, or use `gnu parallel`, whatever floats their boat.

Reply:
I had a go at this, let me know :)